src/i18n/i18n.ts

```diff
@@ -72,7 +72,7 @@ const DOJO_PATH = 'dojo/{bundle}/lookup';
 
 let supportedLocales: string[] = [];
 let defaultLocale = '';
-let computedLocale = '';
+let computedLocale = 'unknown';
```

Comment:
Happy to go with `default` / `unset` / `unspecified` or even use the browsers default i.e. `global.navigator.language || global.navigator.userLanguage`

This is only going be used when there is no `locale` set in the `.dojorc` and therefore be used for loading the default messages from nls bundles.

Reply:
I'm fine with `unknown`, but a quick comment explaining its purpose would be nice.